### DISCUSSION TITLE
Pickup switch

### DISCUSSION THREAD

**Discussion by jenarehorka** [PricelessToolkit/ESPBell-LITE#3] [Q&A]
Hello, this project came in the right time for me as i just moved and want to make my in appartment Fermax intercom smart.

I was about to bring my own esp32 solution but I'm definately going to buy your board. 

My worri is how to handle that on my phone is necessary to pick up the phone so the momentary swich is open. 

Pardon if i missed it somwhere i'm there might be an solution.

I already ordered a push/pull solenoid as want to start experimenting with it.

Thank you so much for your help

Jan

**Comment by PricelessToolkit** [comment]
Hi, sorry but I don't understand this part "My worry is how to handle that on my phone is necessary to pick up the phone so the momentary switch is open." ESPBell has a solid state relay so you can control loads like "solenoid" software part is done in "home assistant". I recommend watching the entire video so you can understand ESPBell's capabilities.

**Reply by jenarehorka** [reply]
Appologies if I'm unclear.

My phone's board have the switch that is checking if the headset is picked up or hangned. If hanged you can not open the house doors. So the opening sequence have to start with pick-up of the phone.

My question is if you have any experience or advice how to solve this. Or if the solenoid push/pull to push the headset slighly away of the base is the only way. Thank you

I watched the video. I like it:)

**Reply by PricelessToolkit** [reply]
Ah ok, if you are 100% sure that if you push the open button when the headset is hung it will not open the door?
1. if it can't open when hung, then you can use a "normally closed signal relay" and pass a switch wire that checks if the headset is picked through the relay. After the doorbell is detected first power up the relay then open the door.

**Reply by mattiasbrandt** [reply]
> So the opening sequence have to start with pick-up of the phone.

I have similar issue on my Aiphone intercom, I need to trigger a "talk" switch before I can trigger the "lock" switch.
Hoping we can solve this with the extra GPIO on the ESPBell-LITE

*subscribing*

**Reply by PricelessToolkit** [reply]
Yes, it can be done easily, you just need a relay board that can take the GPIO signal.

**Reply by mattiasbrandt** [reply]
Any suggestion as to what kind of relay I would need for this? Really appreciate the help

**Reply by PricelessToolkit** [reply]
Something that works with 5v, if size does not matter, use a regular relay board from Ali. https://www.aliexpress.us/w/wholesale-small-relay-board-5v.html     like this one https://www.aliexpress.us/item/1005006233156967.html
